tasks/feeCollection.ts

```diff
@@ -0,0 +1,154 @@
+import "@nomiclabs/hardhat-ethers";
+import { task } from "hardhat/config";
+import mustache from "mustache";
+import path from "path";
+import { getConfig } from "../deployConfig/config";
+import { getPool } from "../poolConfigs/pool-addresses/pools";
+import { FeeCollector, MarginEngine, VAMM } from "../typechain";
+
+interface MultisigTemplateData {
+  poolsCollection: {
+    marginEngineAddress: string;
+    amount: string;
+  }[];
+  feeCollectorAddress: string;
+  distributeAssets: boolean;
+  collectDefaultFund: boolean;
+  collectProtocolFees: boolean;
+  assets: string[];
+  multisig: string;
+  chainId: string;
+}
+
+function writeUpdateTransactionsToGnosisSafeTemplate(
+  data: MultisigTemplateData
+) {
+  // Get external template with fetch
+  const fs = require("fs");
+  const template = fs.readFileSync(
+    path.join(__dirname, "templates/collectFees.json.mustache"),
+    "utf8"
+  );
+  const output = mustache.render(template, data);
+
+  const file = `tasks/JSONs/collectFees.json`;
+  fs.writeFileSync(file, output);
+}
+
+// Description:
+//   This task generates mutiple fee collection transactions
+//   if the --multisig flag is set. Otherwise, it executes the transaction.
+//
+// Example:
+//   ``npx hardhat collectFees --network mainnet --collect --distribute --transfer-protocol-fee --transfer-default-fund --multisig aUSDC_v1 aUSDC_v2``
+//
+task("collectFees", "Set pausability state")
```

Comment:
update description

Reply:
updated in [cbe77ae](https://github.com/Voltz-Protocol/voltz-core/pull/266/commits/cbe77ae66a92b88405421de1c72b1ee0227276a1)